webui/src/components/Header/Header.tsx

```diff
@@ -67,14 +67,23 @@ const DisabledTabWithTooltip = (props: TabProps) => {
 function Header() {
   const classes = useStyles();
   const location = useLocation();
-  const [selectedTab, setTab] = React.useState(location.pathname);
 
-  const handleTabClick = (
-    event: React.ChangeEvent<{}>,
-    newTabValue: string
-  ) => {
-    setTab(newTabValue);
-  };
+  // Prevents error of invalid tab selection in <Tabs>
+  function highlightTab() {
+    switch (location.pathname) {
+      case '/':
+        return '/';
+      case '/code':
+        return '/code';
+      case '/pulls':
+        return '/pulls';
+      case '/settings':
+        return '/settings';
+      default:
+        // using false as value for tabs will result in no selected tab
+        return false;
+    }
+  }
```

Comment:
```suggestion
  function highlightTab() {
    switch (location.pathname) {
      case '/':
      case '/code':
      case '/pulls':
      case '/settings':
        return location.pathname;
      default:
        // using false as value for tabs will result in no selected tab
        return false;
    }
  }
```

What about this?

Reply:
Makes sense. I think I could even remove the switch... Will be back soon.

Reply:
This is even more [compact](https://github.com/MichaelMure/git-bug/pull/625/commits/f980d9487ed1e764a364d5a1f385a09835276962#diff-65c829cfa57d5761b963b25ffac663098388927e2ac8abc286e261c8693f1be5R74-R76) :)
What do you think?